inheritance/lib/src/test/java/inheritance/LibraryTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package inheritance;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LibraryTest {
+
+    @Test
+    void testRestaurant1()
+    {
+        Restaurant sut = new Restaurant("Tammy's Diner", 10.0);
```

Comment:
Here, you're constructing a Restaurant using your constructor that only accepts name and number of stars. That constructor doesn't have any way of creating a list of reviews, so that will be null until you add it. You could fix this by either:
- creating an empty ArrayList in your test, then using your overloaded constructor to pass in the name, star rating, and empty list, or
- using this same constructor, but adding a line to it that creates and attaches an empty ArrayList when the constructor is called